libethereum/Executive.cpp

```diff
@@ -89,7 +84,8 @@ void StandardTrace::operator()(uint64_t _steps, Instruction inst, bigint newMemS
 		m_lastInst.push_back(inst);
 		newContext = true;
 		m_levels.append(std::to_string(_steps));
-		r["levels"] = m_levels;
+		r["levels"] = m_levels; // TODO: move the processing to JavaScript.
+		r["calldata"] = memDump(ext.data.toBytes(), 16);
```

Comment:
Please don't use `memDump` (it performs some strange formatting), but rather use `toHex` as we use for the memory below.